src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -308,7 +383,7 @@ private function addProductVariantToOrder(ProductVariantInterface $productVarian
      */
     private function createOrder(
         CustomerInterface $customer,
-        $number,
+        $number = null,
```

Comment:
Nope, order gets number when the checkout is completed, I use this method to create cart orders.